src/ngraph_builder.cc

```diff
@@ -572,7 +562,8 @@ tf::Status Builder::TranslateGraph(const std::vector<tf::TensorShape>& inputs,
     else if (op->type_string() == "ConcatV2") {
       if (op->num_inputs() < 2) {
         return tf::errors::InvalidArgument(
-            "Number of inputs is not at least 2 for ConcatV2");
+            op->name(), " requires at least 2 inputs, got ", op->num_inputs(),
```

Comment:
Please consider adding another ValidateInputCount(...) which takes a min and max and returns error if the input count is not within the range. Then we can use that here and other ops where there are at least N inputs needed or less than M inputs. Can you think of a lambda as a predicate? Or just use simple overloads.

Reply:
I'll go with simple overloads, because the error message is different.